lib/ansible/constants.py

```diff
@@ -101,14 +101,15 @@ def __getitem__(self, y):
 REJECT_EXTS = ('.pyc', '.pyo', '.swp', '.bak', '~', '.rpm', '.md', '.txt', '.rst')
 BOOL_TRUE = BOOLEANS_TRUE
 COLLECTION_PTYPE_COMPAT = {'module': 'modules'}
+DOC_EXTENSIONS = ('.py', '.yml')
```

Comment:
And yaml (as per PR description)?

Reply:
previous feedback was against .yaml being added

Reply:
Thanks for confirming, could you please update PR description (and possibly commit message then?)